log.capnp

```diff
@@ -1961,6 +1961,11 @@ struct EncodeData {
 struct UserFlag {
 }
 
+struct Microphone {
+  noiseLevel @0 :Float32;
+  noiseLevelRaw @1 :Float32;
```

Comment:
would add "ambient" prefix and "db" suffix

Reply:
also add a comment saying the other one is filtered

Reply:
I'm not sure it actually is dB